data/discord_messages.yaml

```diff
@@ -393,22 +346,8 @@ tasmota:
 
     They have their own [Discord server](https://discord.gg/Ks2Kzd4>) or you can
     ask in <#444529271806099486>
-python:
-  content: >-
-    Python 3.9 is the minimum version of Python supported by Home Assistant,
-    with 3.9 being recommended. If you're using a Python virtual environment
-    install you can install an updated version of Python, then manually upgrade
-    your venv as described in
-    <https://blog.ceard.tech/2017/12/upgrading-python-virtual-environment.html>
```

Comment:
Looks like I can't add suggestions on deleted lines, this section I think we should keep.